python/pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[build-system]
```

Comment:
As mentioned, this is now explicitly added, even though it is generated.

It's perhaps mildly duplicating; and _could_ be worked around, I think, if we really dislike it.

Reply:
I'm going to fix this actually.